master/internal/kubernetes/events.go

```diff
@@ -77,3 +80,25 @@ func (e *eventListener) startEventListener(ctx *actor.Context) error {
 
 	return nil
 }
+
+func (e *eventListener) modMessage(msg *k8sV1.Event) {
+	replacements := map[string]string{
+		"Insufficient nvidia.com/gpu": "Waiting for resources. ",
+		"pod triggered scale-up":      "Job requires additional resources, scaling up cluster.",
+		"Successfully assigned":       "Cluster scaled up, pod resources allocated",
```

Comment:
question: does `Successfully assigned` always mean the cluster scaled up?

Reply:
I guess it doesn't always mean that - it just means the pod was able to be assigned by the scheduler. Will change this.